storage-providers/filecoin-deals/filecoin-programs.md

```diff
@@ -10,40 +10,65 @@ Although it is possible to find your own data storage customers with valuable da
+[![GitHub Repo stars](https://img.shields.io/github/stars/filecoin-saturn/L1-node)](https://github.com/filecoin-saturn/L1-node)
+![GitHub last commit](https://img.shields.io/github/last-commit/filecoin-saturn/L1-node)
+[![Read the doc](https://img.shields.io/badge/docs-saturn.tech-blue)](https://doc.saturn.tech)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C03DH0BL02E)
 
-See the below video for more information on Slingshot:
+A whole new access capability is being launched with [Saturn](https://saturn.tech), which is the world’s first Web3 Content Delivery Network (CDN). Saturn, as a fully distributed CDN, allows clients to access their data via Saturn nodes close to them. Content retrieval times of less than 1 second will open up various new use-cases for Filecoin and create a new market for retrieval providers (L1 Saturn nodes) in which storage providers can also participate.
 
-{% embed url="https://www.youtube.com/watch?v=14sasiEThig" %}
-A overview of the Slingshot project from the Enterprise Storage Provider Accelerator (ESPA).
-{% endembed %}
+## Singularity
 
-## Evergreen
+[![Github Repo stars](https://img.shields.io/github/stars/data-preservation-programs/singularity)](https://github.com/data-preservation-programs/singularity)
+![GitHub last commit](https://img.shields.io/github/last-commit/data-preservation-programs/singularity)
+[![Read the doc](https://img.shields.io/badge/docs-gitbook.io-blue)](https://data-programs.gitbook.io/singularity)
+[![Join Slack](https://img.shields.io/badge/join-Slack-purple)](https://filecoinproject.slack.com/archives/C05JABREATH)
 
-[Evergreen](https://evergreen.filecoin.io/) extends the Slingshot program by aiming to store the open datasets forever. Standard deals have a maximum duration of 540 days, which is not long enough for valuable, open datasets that might need to be stored forever. Evergreen uses a deal engine ([Spade](https://github.com/ribasushi/spade)) that automatically renews deals to extend the lifetime of the dataset on-chain.
+[Singularity](https://singularity.storage) is an end-to-end solution for onboarding datasets to Filecoin storage providers, supporting [PiB-scale data](https://stats.singularity.storage/). It offers modular compatibility with various data preparation and deal-making tools, allowing efficient processing from local or remote storage. Singularity integrates with over 40 storage solutions and introduces inline preparation, which links CAR files to their original data sources, preserving dataset hierarchies. It also supports content distribution and retrieval through multiple protocols and provides push and pull modes for deal making along with robust wallet management features.
 
-## Saturn
+## Evergreen
```

Comment:
evergreen moves to the deprecated section, as the https://evergreen.filecoin.io/ site is going away. Spade the tool is not going away, and should  get its own block in the live section.
Also should have an April 2024 closure date, along with the other deprecated items.
Also talk about deprecated programs in the past tense...